templates/cbt/NE/templates.yaml

```diff
@@ -0,0 +1,43 @@
+dataset: cbt
+subset: NE
+templates:
+  0725fe5e-1bba-4e08-a448-9e0038164914: !Template
+    id: 0725fe5e-1bba-4e08-a448-9e0038164914
+    jinja: '{{sentences | join('''')}}
+
+
+      Write the next sentence of this story.
+
+      |||
+
+      {{ question.replace("XXXXX", answer) }}'
+    name: next_sentence_generation
+    reference: Generate the next sentence given the story.
+  2c326181-dbba-401e-accb-d84ea0162f0a: !Template
+    id: 2c326181-dbba-401e-accb-d84ea0162f0a
+    jinja: "{{ sentences | join('') }}
+
+      {{question}}
+
+      Which part-of-speech fill replaces XXXXX the best?
+
+      |||
+
+      Proper Noun"
```

Comment:
This task I thought could be something like PoS prediction and the model should have some minimal PoS knowledge? Otherwise we can just put the answer word here without any options? Wdyt?